### PR TITLE
METRICS-1: Add util method to build event object

### PR DESCRIPTION
**Pull request by Akayeshmantha**
### Description of what I changed
Introduce method to build the Event object.

**Issue I worked on**
see :- https://issues.openmrs.org/browse/METRICS-1

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/metrics/api/utils/EventsUtils.java [hunk 1]
```diff
@@ -57,4 +70,41 @@ public static String readResourceFile(String file) throws MetricsException {
...
+	
+	public static MetricEvent buildEventObject(String simpleName, String uuid, Event.Action eventAction,
+	        EventConfiguration eventConfiguration) {
+		if (!eventConfiguration.isEnabled()) {
+			LOGGER.debug("Skipped writing '{}' Event to the database because "
+			        + "the synchronization for this object is disabled in the configuration", simpleName);
+			return null;
+		}
+		
+		StringBuilder tags = new StringBuilder(eventAction.name());
```
Comment (ibacher) [inline api/src/main/java/org/openmrs/module/metrics/api/utils/EventsUtils.java]: Do we need a StringBuilder if this is only going to be a single value?

Comment (Akayeshmantha) [reply api/src/main/java/org/openmrs/module/metrics/api/utils/EventsUtils.java]: yes true

**Comment by Akayeshmantha** [comment]
@ibacher  pushed the changes.

**Review by ibacher — APPROVED** [review]

**Comment by ibacher** [comment]
@Akayeshmantha I've merged this. Please look at how I've updated the ticket title and commit message. Commits should, ideally, be in the format: <PROJECT>-<TICKET #>: <brief description>.

**Pull request merged**